src/service/TodoService.ts

```diff
@@ -0,0 +1,25 @@
+import { Todo } from "../interfaces/Todo";
+
+export const TodoService = {
+  getTodos(): Todo[] {
+    const todosJSON = localStorage.getItem('todos');
+    if (todosJSON) {
+      return JSON.parse(todosJSON);
+    }
+    return [];
+  },
+
+  setTodos(todos: Todo[]) {
+    const todosJSON = JSON.stringify(todos);
+
+    localStorage.setItem('todos', todosJSON);
+  },
+
+  addTodo(title: string): Todo {
+    const todos = this.getTodos();
+    const newTodo: Todo = { id: todos.length + 1, title, done: false };
```

Comment:
It can be problematic, because if some todos were deleted, the length of todos is not equal to the highest id + 1! Please reconsider it!